src/main/scala/cromwell/engine/SymbolStore.scala

```diff
@@ -26,9 +26,47 @@ class SymbolStore(binding: WdlBinding, inputs: Map[FullyQualifiedName, WdlValue]
     assignSymbolStoreEntry(fullyQualifiedName, value, input = true)
   }
 
+  for {
+    workflow <- binding.workflows
+    call <- workflow.calls
+    (k, v) <- call.inputMappings
+  } yield assignSymbolStoreEntry(s"${call.fullyQualifiedName}.$k", v, input = true)
+
+  private def callEntries(call: Call): Set[SymbolStoreEntry] =
+    store.filter {entry => entry.scope == call.fullyQualifiedName}.toSet
```

Comment:
My personal take is that if the function is going to span multiple lines that you should use curly braces. If it's all on one line, then eschewing the curly braces is the right thing to do.